### PR TITLE
added put request

### PR DESCRIPTION
**Pull request by Skazka123**

### CODE REVIEW

File: models/libraryItems/libraryItemAPI.js [hunk 1]
```diff
@@ -44,17 +45,33 @@ const getPendingItems = () => libraryItem.find({ approved: false });
 const acceptPendingItem = id =>
   libraryItem.findById(id).then(item => {
     item.set({ approved: true });
-    item.save();
+    return item.save();
```
Comment (demns) [inline models/libraryItems/libraryItemAPI.js]: does save return anything?

File: models/news/newsAPI.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 const News = require('./news');
-const isValidObjectId = require('./isValidObjectId');
+const isValidObjectId = require('../isValidObjectId');
```
Comment (demns) [inline models/news/newsAPI.js]: great, now can we move it to utils folder or somewhere?

Comment (Skazka123) [reply models/news/newsAPI.js]: done

File: models/libraryItems/libraryItemAPI.js (hunk 1 above)
Comment (Skazka123) [reply models/libraryItems/libraryItemAPI.js]: I guess we don't need it here. Deleted it.

File: routes/libraryRoutes.js [hunk 3]
```diff
@@ -45,6 +47,11 @@ const libraryRoutes = router => {
   });
```
Comment (Kukharchuk) [inline routes/libraryRoutes.js]: spaaaace

File: routes/libraryRoutes.js [hunk 4]
```diff
@@ -45,6 +47,11 @@ const libraryRoutes = router => {
...
   router
     .route('/library/:_id')
+    .get((req, res) => {
+      getItemById(req.params._id).then(item => {
+        res.json(item);
+      });
```
Comment (Kukharchuk) [inline routes/libraryRoutes.js]: ggggg

**Pull request merged**